pkg/products/monitoring/prometheusRules.go

```diff
@@ -292,6 +292,22 @@ func (r *Reconciler) newAlertsReconciler() resources.AlertReconciler {
 					},
 				},
 			},
+			{
+				AlertName: "install-upgrade-alerts",
+				Namespace: r.Config.GetOperatorNamespace(),
+				GroupName: "general.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: "RHMICSVAbnormal",
```

Comment:
Lets call it `RHMICSVRequirementsNotMet`

Reply:
Noted, will push the change soon